src/core_modules/capture-core/components/Pages/MainPage/EventWorkingLists/Redux/EventWorkingListsRedux.container.js

```diff
@@ -9,35 +9,36 @@ import {
 import { EventWorkingListsColumnSetup } from '../ColumnSetup';
 import { useWorkingListsCommonStateManagement } from '../../WorkingListsCommon';
 import { getEventProgramThrowIfNotFound } from '../../../../../metaData';
+import { SINGLE_EVENT_WORKING_LISTS_TYPE } from '../constants';
 import type { Props } from './eventWorkingListsRedux.types';
 
-export const EventWorkingListsRedux = ({ listId, ...passOnProps }: Props) => {
+export const EventWorkingListsRedux = ({ storeId, ...passOnProps }: Props) => {
```

Comment:
This is perhaps not relevant to this PR but after our conversation it looked like passOnProps here arent carrying anything with them